packages/fxa-dev-launcher/README.md

```diff
@@ -12,7 +12,7 @@ Available options:
 - `FIREFOX_DEBUGGER=true` - open the [Browser Toolbox](https://developer.mozilla.org/en-US/docs/Tools/Browser_Toolbox) on start (NOTE: `false` by default for speed).
 - `FXA_DESKTOP_CONTEXT` - context value for the fxa-content-server: `context=[value]` (NOTE: `fx_desktop_v3` is default).
```

Comment:
Not related to your patch directly but the property you are documenting: there is one other `FXA_DESKTOP_CONTENT` doc line for this property (line 10) which is incorrect and duplicated by this one.

Reply:
I will get this modified in another quick PR to avoid CI rerun here, thanks!